tests/DialogServiceConnectorTests.ts

```diff
@@ -329,6 +339,7 @@ test("ListenOnceAsync", (done: jest.DoneCallback) => {
         });
 
     WaitForCondition(() => (recoCounter === 2), done);
+    WaitForCondition(() => (turnStatusCounter == 1), done);
```

Comment:
```
WaitForCondition(() => (turnStatusCounter === 1), done);
```

Reply:
Very astute catch -- manual re-run of the test pipeline after pushing got it, too:
```
2021-01-12T18:21:07.6639690Z ERROR: D:/a/1/s/tests/DialogServiceConnectorTests.ts:319:33 - == should be ===
2021-01-12T18:21:07.6641862Z ERROR: D:/a/1/s/tests/DialogServiceConnectorTests.ts:342:47 - == should be ===
```

I hadn't configured local testing yet as I wasn't sure how to do it. It's not particularly easy without the pipeline automating secrets (and I don't think everyone will have all the things available/configured), but I'll add a blurb to the readme to help future contributors, too.